File: RealEstate-Source-Code-v6.0/AdminPanel-Laravel10/app/Http/Controllers/UploadController.php
<?php

namespace App\Http\Controllers;

use App\Library\FileHelper;
use Illuminate\Http\Request;
use Illuminate\Support\Facades\File;
use Illuminate\Support\Str;

class UploadController extends Controller
{
    public function image_get(Request $request)
    {
        $load = $request->load;
        if (FileHelper::checkImageFile($load)) {
            return redirect('/uploadfiles/images/' . $load);
        }
        return redirect('/uploadfiles/caches/' . $load);
    }

    public function image_create(Request $request)
    {
        $imageUploaded = request()->file('filepond');
        $imageName = Str::uuid()->toString() . '.' . $imageUploaded->getClientOriginalExtension();
        $imagePath = public_path('/uploadfiles/caches');
        $imageUploaded->move($imagePath, $imageName);
        return $imageName;
    }

    public function image_delete(Request $request)
    {
        $imageName = request()->getContent();
        $imagePath = public_path('/uploadfiles/caches/');
        File::delete($imagePath . $imageName);
        return $imageName;
    }
}

File: RealEstate-Source-Code-v6.0/AdminPanel-NetCore/Controllers/UploadController.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Threading.Tasks;
using Microsoft.AspNetCore.Http;
using Microsoft.AspNetCore.Mvc;
using RealEstate.Helpers;

namespace RealEstate.Controllers
{
    [Route("[controller]")]
    public class UploadController : ControllerBase
    {
        [HttpGet("Image")]
        public IActionResult Get(string load)
        {
            if (FileHelper.checkImageFile(load))
            {
                return Redirect("~/UploadFiles/Images/" + load);
            }
            else
            {
                return Redirect("~/UploadFiles/Caches/" + load);
            }
        }

        [HttpPost("Image")]
        public async Task<IActionResult> Post(IFormFile filepond)
        {
            var fileName = Guid.NewGuid().ToString() + Path.GetExtension(filepond.FileName);
            var filePath = Path.Combine(Directory.GetCurrentDirectory(), @"UploadFiles\Caches", fileName);

            using (var stream = System.IO.File.Create(filePath))
            {
                await filepond.CopyToAsync(stream);
            }

            return Ok(fileName);
        }

        [HttpDelete("Image")]
        public async Task<IActionResult> Delete()
        {
            using (var reader = new StreamReader(Request.Body))
            {
                var fileName = await reader.ReadToEndAsync();
                var filePath = Path.Combine(Directory.GetCurrentDirectory(), @"UploadFiles\Caches", fileName);
                System.IO.File.Delete(filePath);
            }
            return Ok();
        }
    }
}